dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First2.json

```diff
@@ -13,6 +13,12 @@
       "Still! Even if you come back now, there's no guarantee you will next time!",
       "Are you still going despite knowing that?!"
     ],
+    "pl": [
+      "Haa, to jesteś z nimi teraz, Stacju?\nI nie jesteś tu, aby zostać?",
+      "Łał...",
+      "I tak! Tylko przez tego że teraz wróciłaś, nie oznacza że na pewno wrócisz następnym razem!",
+      "Czy dalej idziesz mimo wiedząc tego?!"
```

Comment:
"Dalej chcesz iść wiedząc to?" Changed the sentence a little bit, because I think it sounds a bit better like that